packages/token-bridge-sdk/src/withdrawals/__test__/fetchETHWithdrawalsFromEventLogs.test.ts

```diff
@@ -18,69 +22,45 @@ describe('fetchETHWithdrawalsFromEventLogs', () => {
 
   it('fetches some ETH withdrawals from event logs pre-nitro', async () => {
     const result = await fetchETHWithdrawalsFromEventLogs({
-      address: '0x41C966f99De0cA6F6531fbcAc9Db7eaBDF119744',
-      fromBlock: 11110000,
-      toBlock: 12055296,
+      address: l2UserAddress,
+      fromBlock: 20785772,
+      toBlock: 22964111,
```

Comment:
why is this block number used?
nitro genesis block is `22207817`
so we should use `22207817 - 1` for the last block pre-nitro?

`22964111` includes post nitro blocks

Reply:
Yup, but we don't need the nitro genesis block, we want a range that starts pre-nitro and ends post-nitro so we can make sure the subgraph returns both!